Makefile

```diff
@@ -25,29 +25,31 @@ endif
 unit: ## Run unit tests against code.
 ifeq ($(CONTAINER_RUNNABLE), 0)
 		$(CONTAINER_RUNTIME) run -it -v ${PWD}:/go/src -w /go/src docker.io/library/golang:${GO_VERSION}-alpine3.17 \
-         /bin/sh -c "go test ./... -v -coverprofile ${TEST_COVERAGE_FILE}; \
+         sh -e ./run-for-all-go-modules.sh "go test ./... -v -coverprofile ${TEST_COVERAGE_FILE}; \
```

Comment:
More typical approach I have seen is a list of subdirs in each Makefile and then looping through that and calling `make test` on each. But this is fine too. Also seems like there is probably a more "make native" way, but this works.

Reply:
I am not a Makefile guru either, so I am pretty sure that there is a more make-native way :)

If all of my higher priority tasks are solved, I will try to put the list of subdirs into the Makefile and call make recursively, as you've suggested. I would still prefer to assemble the list dynamically (by finding go.mod files) as opposed to have a manually maintained list